common/etc/adminhtml/routes.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:App/etc/routes.xsd">
     <router id="admin">
-        <route id="webwinkelkeur" frontName="webwinkelkeur">
-            <module name="WebwinkelKeur_Magento2" />
+        <route id="{{ LOWERCASE_PROJECT }}" frontName="{{ LOWERCASE_PROJECT }}">
+            <module name="{{ PROJECT }}_Magento2" />
```

Comment:
Import controller cannot be found, https://dev.magento.com/admin/trustprofile/actions/import/store/0/ returns `404`
This only works if `Import.php` is under the namespace `namespace TrustProfile\Magento2\Controller\Adminhtml\Actions`.

I thought a solution would be to make this `<module name="Valued_Magento2" />` but the current module name is 'WebwinkelKeur_Magento2` or `TrustProfile_Magento2`